detroit-ui-components/src/locales/general.json

```diff
@@ -1593,5 +1593,6 @@
   "publicFilter.waitlist.open": "Open Waitlist",
   "publicFilter.waitlist.closed": "Closed Waitlist",
   "account.noFavorites": "It looks like you haven't favorited any listings yet.",
-  "srNewPage": "New Page"
+  "srNewPage": "New Page",
+  "srPageTitle": "Page Title "
```

Comment:
Concatenating strings like this isn't best practice because other languages might not order the strings this way, and they'll just be translated as standalone. A workaround here is to make the string "Page Title:" so it's clear to translators that the page title is coming next in the string.